### DISCUSSION TITLE
How to relocate a EME installation from one WP-Instance to another?

### DISCUSSION THREAD

**Discussion by Schneleed** [liedekef/events-made-easy#587] [Q&A] [answered]
Dear community,

I am planning to set up a new WP-Instance and want to move my current EME-installation from my old WP to the new one.
Is there any doc or howto how this is made without losing data?

Thanks in advance for your help.

**Comment by liedekef** [comment, marked as answer]
EME consists of 2 parts: the database tables and wordpress options.
The relevant database tables can be exported/imported using your favorite tool (the names of the tables start with "wp_eme_", with "wp_" being your database prefix of choice)
The wordpress options are in the wordpress options table, you can search for those with names starting with "eme_" and export those, then import those in the wordpress options table of your new setup.
When all that is done, you can then install EME on your new setup and it will just work.
